rio_tiler_pds/landsat/utils.py

```diff
@@ -129,32 +183,49 @@ def sceneid_parser(sceneid: str) -> Dict:
     elif meta["sensor"] == "M":
         sensor_name = "mss"
 
+    # S3 paths always use oli-tirs
+    _sensor_name = sensor_name
+    if _sensor_name in ["oli", "tirs"]:
+        _sensor_name = "oli-tirs"
```

Comment:
are we sure about this?

Reply:
This is what I'm referring to. There's no `oli` or `tirs` folder here
```
> aws s3 ls s3://usgs-landsat/collection02/level-1/standard/ --request-payer
                           PRE etm/
                           PRE mss/
                           PRE oli-tirs/
                           PRE tm/
2020-08-26 19:35:23       1313 catalog.json
```

Reply:
yes, but you also said you didn't find any `oli` only scenes, right ?

Reply:
I did, and I added them to the tests
https://github.com/cogeotiff/rio-tiler-pds/pull/42/files#diff-be27e3477e1b5abde00cf4e303a5eeec6d0f57689c97a9f9c7468fe48262e3d8R178-R227

Plus a `tirs`-only scene
https://github.com/cogeotiff/rio-tiler-pds/pull/42/files#diff-be27e3477e1b5abde00cf4e303a5eeec6d0f57689c97a9f9c7468fe48262e3d8R278-R302